packages/context/src/inject.ts

```diff
@@ -299,10 +300,23 @@ export function describeInjectedArguments(
   method?: string,
 ): Readonly<Injection>[] {
   method = method || '';
+  const options: InspectionOptions = {};
+  if (method === '') {
+    // A hacky way to check if an explicit constructor exists
+    // See https://github.com/strongloop/loopback-next/issues/1565
+    if (target.toString().match(/\s+constructor\s*\([^\)]*\)\s+\{/m)) {
+      options.ownMetadataOnly = true;
+    }
+  } else if (target.hasOwnProperty(method)) {
```

Comment:
Is this code block covered in a test?

Reply:
According to Coveralls, this branch is executed several times (14x) by our test suite. However, I cannot tell if there is a test that would fail if this branch was not present.

@raymondfeng could you please add an explicit test for this branch?